modules/end-user-guide/partials/proc_setting-up-che-editor-yaml.adoc

```diff
@@ -0,0 +1,14 @@
+[id="setting-up-che-editor-yaml_{context}"]
+= Setting up che-editor.yaml
+
+To define the same in-browser IDE for all workspaces that will clone the same remote Git repository of your project, you can use the `che-editor.yaml` file. Using this file allows you to set a common default editor for your team and to provide new contributors with a default editor that is most suitable for your project. You can also use the `che-editor.yaml` file when you need to set a different IDE default for a particular repository of your project rather than use the default IDE of your organization's {prod-short} instance.
```

Comment:
```suggestion
To define the same in-browser IDE for all workspaces that will clone the same remote Git repository of your project, you can use the `che-editor.yaml` file. This way, you can set a common default editor for your team and provide new contributors with the most suitable editor for your project. 

You can also use the `che-editor.yaml` file when you need to set a different IDE default for a particular project repository rather than the default IDE of your organization's {prod-short} instance.
```